Content.Shared/Pinpointer/SharedNavMapSystem.cs

```diff
@@ -140,20 +145,33 @@ protected bool TryCreateNavMapBeaconData(EntityUid uid, NavMapBeaconComponent co
 
     private void OnGetState(EntityUid uid, NavMapComponent component, ref ComponentGetState args)
     {
```

Comment:
Why is & was this cloning all of the chunk data instead of just using the existing dictionaries?